src/js/ads.js

```diff
@@ -326,9 +326,11 @@ class VideoAds {
 	}
 
 	adErrorHandler(adError) {
+		// NOTE: has the API changed? now need to call `getError` method to get the ad error
+		const actualError = ('getError' in adError) ? adError.getError() : adError;
```

Comment:
This line will throw if `getError` is not a function. If we include `&& typeof adError.getError === 'function'` it will protect us from this scenario.